index.html

```diff
@@ -47,19 +40,28 @@
       </div>
     </nav>
     <main class="main-container">
-      <h1 class="hidden-heading">Homepage Hidden Heading</h1>
-      <h2 class="main-heading">
-        Unleash Your Creativity to Utmost
-        <span class="text-highlight">Potential</span> with Us
-      </h2>
-      <h3 class="main-subheading">
-        Discover the building blocks of your next masterpiece: premium digital
-        assets at your fingertips.
-      </h3>
-      <div class="search-container">
-        <p class="main-text">Find fonts, templates, illustrations, and more…</p>
-        <a href="#" class="main-search-btn">Search Now</a>
-      </div>
+      <section class="hero-container">
+        <h1 class="hidden-heading">Homepage Hidden Heading</h1>
+        <h2 class="main-heading">
+          Unleash Your Creativity to Utmost
+          <span class="text-highlight">Potential</span> with Us
+        </h2>
+        <h3 class="main-subheading">
```

Comment:
All block content cannot consist only of headings. A heading should title some content. In this block, level 3 heading do not title anything and therefore are not headings.

Subheadings should not be made with `<h>` elements (better use `<p>`). Details: https://www.w3.org/TR/html52/common-idioms-without-dedicated-elements.html#subheadings-subtitles-alternative-titles-and-taglines